spec/features/articles/user_adds_article_spec.rb

```diff
@@ -56,15 +56,15 @@
     expect(page).to have_content "Description is too long (maximum is 500 characters)"
   end
 
-scenario "User enters title that is too long" do
-  sign_in_as(user)
+  scenario "User enters title that is too long" do
+    sign_in_as(user)
 
-  visit new_article_path
-  fill_in 'Name', with: 'Here is a title that is over 50 characters. Much too long dude! Do not be so long! You are annoying for everyone!'
+    visit new_article_path
+    fill_in 'Name', with: 'Here is a title that is over 50 characters. Much too long dude! Do not be so long! You are annoying for everyone!'
 
-  click_button 'Submit'
+    click_button 'Submit'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.